Program.cs

```diff
@@ -257,43 +259,82 @@ public static void CombineAtlas(string inputPath)
 
         static void Main(string[] args)
         {
-            string input = string.Empty;
+            string input  = "";
             string output = "";
             bool extract = false;
             bool combine = false;
             for (int i = 0; i < args.Length; ++i)
             {
                 var arg = args[i];
-                if (arg == "--input" || arg == "-i")
-                {
-                    i++;
-                    input = args[i].Trim();
-                }
-                else if (arg == "--output" || arg == "-o")
-                {
-                    i++;
-                    output = args[i].Trim();
-                }
-                else if (arg == "--export" || arg == "-x")
-                {
-                    extract = true;
-                }
-                else if (arg == "--combine" || arg == "-c")
-                {
-                    combine = true;
-                }
-                else if (arg == "--help" || arg == "-h")
-                {
-                    Console.WriteLine($"Atlas Tool \n");
-                    Console.WriteLine($"Usage [arguments]\n");
-                    Console.WriteLine($"arguments:");
-                    Console.WriteLine($"-i|--input \tFile or folder to extract from");
-                    Console.WriteLine($"-o|--output \tOutput directory to extract into");
-                    Console.WriteLine($"-x|--export \tExport sprites from sprite atlas");
-                    Console.WriteLine($"-c|--combine \tCombine modified sprites into new atlas image");
-                    return;
-                }
+				switch(arg)
```

Comment:
What did you change here? All i can tell is that it was reformatted to use tabs instead of spaces. Please revert back to spaces

Reply:
I'm personally used to tabs instead of spaces, sorry lol, probably just slipped my mind
Most of those were just automatic indentation from VS Code, I didn't pay too much attention to it